.github/workflows/dataflow_engine_e2e.yaml

```diff
@@ -28,10 +28,10 @@ jobs:
           go-version: 1.18
 
       - name: Build images
-        run: $GITHUB_WORKSPACE/engine/sample/prepare.sh
+        run: make engine_build
```

Comment:
May be `make engine_image_build` is more proper. Or it may means `build engine binary` at the first glance.

Reply:
Good advice! Also, how about `engine_image`?